.github/workflows/nixos-manual.yml

```diff
@@ -0,0 +1,19 @@
+name: NixOS manual checks
+
+on:
+  pull_request:
+    branches-ignore:
+      - 'release-**'
+    paths:
+      - 'nixos/**/*.xml'
+      - 'nixos/**/*.md'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2.3.4
```

Comment:
Why `@v2.3.4` and not just `@v2`?

Reply:
I don't have a preference. Does v2 install the latest 2 major version?